docs_src/src/index.html

```diff
@@ -121,6 +179,29 @@
           }
         });
       </script>
+      <!-- Google Tag Manager -->
+    <script>
+      window.dataLayer = window.dataLayer || []
+      function gtag(){dataLayer.push(arguments);}
+
+      if(localStorage.getItem('consentMode') === null) {
+        gtag('consent', 'default', {
+          'ad_storage': 'denied',
+          'ad_user_data': 'denied',
+          'ad_personalization': 'denied',
+          'analytics_storage': 'denied'
+          });
+      } else {
+        gtag('consent', 'default', JSON.parse(localStorage.getItem('consentMode')));
+      }
+      
+    (function(w,d,s,l,i){
+      w[l].push({'gtm.start':
+      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','GTM-W9SKCQ8');</script>
+      <!-- End Google Tag Manager -->
```

Comment:
Do you want to remove the script from here ? Or should I leave it, without the GTM ID ?

Reply:
yes remove it, leave just the gtag function, and initialization there